ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__aws-4.19-nightly-x86-loaded-upgrade-from-4.18.yaml

```diff
@@ -145,6 +145,89 @@ tests:
     - ref: openshift-qe-cluster-density-v2
     workflow: openshift-qe-upgrade
   timeout: 8h0m0s
+- as: loaded-upgrade-418to419-3nodes-udn
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CHURN: "false"
+      ENABLE_FORCE: "true"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "9"
+      RUN_ORION: "false"
+      ZONES_COUNT: "3"
+    post:
+    - ref: openshift-qe-kube-burner-index
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre-stableinitial
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: openshift-qe-udn-density-pods
+    workflow: openshift-qe-upgrade
+  timeout: 7h0m0s
+- as: loaded-upgrade-418to419-24nodes-72udn
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CHURN: "false"
+      ENABLE_FORCE: "true"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "3"
+      MAX_UNAVAILABLE: "3"
+      MAX_UNAVAILABLE_WORKER: "3"
```

Comment:
MAX_UNAVAILABLE_WORKER is from chain: openshift-qe-upgrade-max-unavailable
MAX_UNAVAILABLE is from workflow: openshift-qe-upgrade
You can use either one. If you use the second one, you don't need chain: openshift-qe-upgrade-max-unavailable

Reply:
@SachinNinganure This is not addressed yet